rpcserver.go

```diff
@@ -3373,6 +3378,15 @@ func (r *rpcServer) fetchPendingOpenChannels() (pendingOpenChannels, error) {
 		commitBaseWeight := blockchain.GetTransactionWeight(utx)
 		commitWeight := commitBaseWeight + input.WitnessCommitmentTxWeight
 
+		// FundingExpiryBlocks is the distance from the current block
+		// height to the broadcast height + MaxWaitNumBlocksFundingConf.
+		maxFundingHeight := funding.MaxWaitNumBlocksFundingConf +
+			pendingChan.BroadcastHeight()
+		fundingExpiryBlocks := int32(maxFundingHeight) - currentHeight
+		if fundingExpiryBlocks < 0 {
+			fundingExpiryBlocks = 0
```

Comment:
Maybe we should allow it to be negative?

Reply:
Sure. That does make the comment more meaningful, since it indicates the distance from the current block.